tutorials/chapter2.md

```diff
@@ -274,6 +274,105 @@ int main(int argc, char **argv) {
 * The Environment class interface can be used to get and set values. When getting/setting a parameter, you must know the type of the parameter and use the relevant getter/setter. 
 * After running this example, you can verify that a new parameter `/paramFloat` exists with a value 1.5 using the `rosparam` command-line utility.
 
+#### Example 5: Exchange ROS Messages
+
+This example shows us that with Klepsydra core you can use ROS messages to exchange data between the nodes.
```

Comment:
Add more explanation about what is the difference between this example and the others.